fe/src/main/java/org/apache/doris/planner/BrokerScanNode.java

```diff
@@ -696,9 +699,40 @@ private TBrokerRangeDesc createBrokerRangeDesc(long curFileOffset, TBrokerFileSt
         rangeDesc.setStart_offset(curFileOffset);
         rangeDesc.setSize(rangeBytes);
         rangeDesc.setFile_size(fileStatus.size);
+        rangeDesc.setColumns_from_path(columnsFromPath);
         return rangeDesc;
     }
 
+    private Map<String, String> parseColumnsFromPath(String filePath, List<String> columnsFromPath) throws UserException {
+        if (columnsFromPath == null || columnsFromPath.isEmpty()) {
+            return Collections.emptyMap();
+        }
+        String[] strings = filePath.split("/");
+        Map<String, String> columns = new HashMap<>();
+        for (int i = strings.length - 1; i >= 0; i--) {
+            String str = strings[i];
+            if (str == null || str.isEmpty() || !str.contains("=")) {
```

Comment:
When we meet a path like "/xx/yy/zz/k1=1/aa",  I think we should think this path is not a valid path. We only accept paths like "xx/yy/zz/aa/k1=1"

Reply:
> When we meet a path like "/xx/yy/zz/k1=1/aa", I think we should think this path is not a valid path. We only accept paths like "xx/yy/zz/aa/k1=1"

I will raise an exception here.